### PR TITLE
How to deal with Content-Encoding

### LINKED ISSUE #13: Dealing with multiple encodings
When draft -00 was published and solicited on the WG mailing group list, Roy Fielding [replied](https://lists.w3.org/Archives/Public/ietf-http-wg/2023JanMar/0214.html) with the following:

> Also, you have not considered multiple content codings yet, which would
> (rarely) consist of a private encoding enclosed within a compression:
> 
>    Content-Encoding: bzerp, gzip
> 
> I would expect, in that case, for both encodings to be decoded before the
> identity-digest is applicable. YMMV.

### PR DESCRIPTION
**Pull request by LPardue**
Closes #13

### CODE REVIEW

File: draft-pardue-httpbis-identity-digest.md [hunk 1]
```diff
@@ -207,6 +207,14 @@ Want-Identity-Digest: sha-256=1
 Want-Identity-Digest: sha-512=3, sha-256=10, unixsum=0
 ~~~
 
+# Messages containing both Identity-Digest and Content-Encoding
+
+`Identity-Digest` applies to the unencoded representation. If a message is
```
Comment (mikewest) [inline draft-pardue-httpbis-identity-digest.md:212]: Tiny phrasing nits that read better to me:

```suggestion
Digests delivered through `Identity-Digest` apply to the unencoded representation. If a message is
```

File: draft-pardue-httpbis-identity-digest.md [hunk 2]
```diff
@@ -207,6 +207,14 @@ Want-Identity-Digest: sha-256=1
...
+received with content coding, a recipient needs to decode the message in order
+to calculate the digest that can subsequently be used for validation. If
+multiple content codings are applied, the recipient needs to decode all
+encodings before usage.
```
Comment (mikewest) [inline draft-pardue-httpbis-identity-digest.md:216]:
```suggestion
encodings in order before validation.
```

**Review by mikewest — APPROVED** [review]
LGTM. Small phrasing suggestions; feel free to ignore if the current text is better to your ear.

**Comment by LPardue** [comment]
Great suggestions, thanks.

**Pull request merged**